### PR TITLE
update documentation content

### PR DESCRIPTION
**Pull request by mgazdzik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,6 @@ Please also be aware that it will be required for development team to review cha
 
     ```
     cd _src
-    bin/sculpin install
```
Comment (snapshotpl) [inline README.md]: Don't we need to run this command?

File: _src/source/_docs/use-cases/business-dictionary.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Offer types ##
+
+* **Public** - offer is visible and available for all Trans.eu platform users immediately. Regardless of whether they are employees of the same company, corporation, or whether they belong to a common cluster.
+* **Private** - offer is visible and available only for the employees of the same company as the oferrer.
+* **Cluster** - offer is visible and available to employees of companies that are members of the cluster. The offer becomes visible in cluster right after being added.
+**Note:** a company can be a member of many clusters, and each of them has a separate identifier to be specified during offer adding. 
+For ex.
+ 
+```
```
Comment (snapshotpl) [inline _src/source/_docs/use-cases/business-dictionary.md]: make it correct json and add json annotation

File: _src/source/_docs/use-cases/business.md [hunk 3]
```diff
@@ -998,3 +1015,333 @@ Host: offers.system.trans.eu
...
+  "loading_date": "2016-03-07T07:00:00+0000",
+  "unloading_place": {
+    "address": {
+      "country": "PL",
+      "postal_code": "02-495",
+      "locality": "Warszawa"
+    },
+  },
+  "unloading_date": "2016-03-09T07:00:00+0000",
+...
```
Comment (snapshotpl) [inline _src/source/_docs/use-cases/business.md]: This made this request invalid. Remove it

File: _src/source/_docs/use-cases/business.md [hunk 4]
```diff
@@ -998,3 +1015,333 @@ Host: offers.system.trans.eu
...
+HTTP/1.1 201 Created
+Content-Type: application/hal+json
+
+{
+  "id": 123456790,
+...
+}
+```
+
+###Offer 3###
```
Comment (snapshotpl) [inline _src/source/_docs/use-cases/business.md]: Why three examples?

Comment (mgazdzik) [inline _src/source/_docs/use-cases/business.md]: Use case assumes multiple offers being added, so there are multiple requests as example

**Comment by mgazdzik** [comment]
I generally removed all occurrences of tripple stop (...) but still I wonder if we should mark somehow that given request is NOT a complete request with full field set, or not a full response, just a piece of it showing what's important in given use case?

File: README.md (hunk 1 above)
Comment (mgazdzik) [inline README.md]: actually we should, you're right. but I wonder if we need to do this every time? maybe we should prep a "initialize" script that would prevent install being triggered every time - what do you think?

File: _src/.sculpin/autoload.php [hunk 5]
```diff
@@ -4,4 +4,4 @@
 
 require_once __DIR__ . '/composer' . '/autoload_real.php';
 
-return ComposerAutoloaderInit7482e515ecda47f1caf1ac7bac03151b::getLoader();
+return ComposerAutoloaderInit8a4ff1c5348ed5ec490d58da4182a405::getLoader();
```
Comment (snapshotpl) [inline _src/.sculpin/autoload.php]: Should be keep autoload and other dependencies in repository?

File: README.md (hunk 1 above)
Comment (snapshotpl) [inline README.md]: That's make a sense

**Comment by snapshotpl** [comment]
@mgazdzik we can add at top of page information that's not full responses in examples. That's will ok when some new fields will be added

**Comment by mgazdzik** [comment]
@snapshotpl - implemented your notes, please review

**Pull request merged**